website/src/menu.css.ts

```diff
@@ -53,7 +61,7 @@ export const drawerItem = style([
     display: "inline-block",
     padding: vspace.quarter,
     textDecoration: "none",
-    color: colors.text,
+    color: "black",
```

Comment:
Why this change? I'd rather add another color to the theme that handles this instead of using a hard coded color.

Reply:
There was an odd quirk that made it so menu items were not visible against the background, which has been fixed.